crates/turbopack-dev-server/src/source/resolve.rs

```diff
@@ -113,6 +113,18 @@ async fn request_to_data(
     if vary.body {
         data.body = Some(request.body.clone().into());
     }
+    if vary.raw_query {
+        data.raw_query = Some(request.uri.query().unwrap_or("").to_string());
```

Comment:
🎉. Would you update https://github.com/vercel/turbo/blob/780ff56732564617bbc8da300c335365bf404427/crates/turbopack-node/src/source_map/content_source.rs#L53-L58?